iiitdmj/package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "iiitdmj",
+  "name": "college-website",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "node-sass": "^6.0.1",
```

Comment:
don't remove node-sass

Reply:
node-sass will throw error on build, package have some issues, sass and sass-loader are alternative and works the same.

Reply:
node-sass will not throw the error on the build, as I have used it on my react projects before. It may have an issue on your side as you have deleted a major part of package-lock.json file.

Reply:
deploy it to a azure, heroku or netlify, It will throw error https://stackoverflow.com/questions/68976148/how-to-fix-this-node-sass-build-error-on-netlify/69076580#69076580